Library/Homebrew/warnings.rb

```diff
@@ -0,0 +1,36 @@
+# typed: true
+# frozen_string_literal: true
+
+require "warning"
+
+# Helper module for handling warnings.
```

Comment:
Might want to note this should not e.g. use any of our vendored gems given it needs to work from RuboCop itself outside of Homebrew (right?). Makes me wonder if (not on this PR) we should move all files like that to a special directory 🤔

Reply:
Running just `rubocop` doesn't use this file. Running `brew rubocop` does. Is that what you mean? I don't know if we can avoid requiring this file (and, thus, the `warning` gem) without either showing the warning or adding `-W0` to `brew rubocop`.

Reply:
Misread `Library/Homebrew/utils/rubocop.rb` as `Library/Homebrew/rubocops.rb` above 🤦🏻

Reply:
(ignore me!)